adobe_vipm/adobe/errors.py

```
@@ -81,9 +83,68 @@ def __repr__(self) -> str:
+    """
+    Convert an error response into the matching Adobe error.
+
+    Args:
+        error: The HTTP error raised for the error response.
+
+    Raises:
+        AdobeAPIError: When the response body is the documented Adobe error JSON.
+        AdobeHttpError: When the response body is not JSON.
+    """
+    logger.error(error)
+    try:  # noqa: WPS328, WPS505
```

Ruff (0.16.1)

[warning] Invalid rule code in `# noqa`: WPS328, WPS505

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)